about.html

```diff
@@ -0,0 +1,119 @@
+                <li><a href="#">English</a></li>
+                <li><a href="#">My Page</a></li>
+                <li><a href="#">Logout</a></li>
+            </ul>
+        </div>
+
+        <div class="tab-2">
+            <div class="tab-2-wrapper">
+                <div class="nav">
+                    <div class="hamburger">
+                        <i class="fa-solid fa-bars"></i>
+                    </div>
+                    <div class="logo">
+                        <a href="index.html"><img src="./assets/images/main-images/logo.png" alt=""></a>
+                    </div>
+                </div>
+                <ul class="navbar">
+                    <li class="closebar"><i class="fa-solid fa-xmark"></i></li>
+                    <li><a href="index.html">Home</a></li>
+                    <li><a class="active" href="about.html">About</a></li>
+                    <li><a href="#">Program</a></li>
+                    <li><a href="#">Join</a></li>
+                    <li><a href="#">Sponsor</a></li>
+                    <li><a href="#">GL Campaign</a></li>
+                </ul>
+            </div>
+        </div>
+    </header>
+    <main>
+        <section class="hero-section">
+            <div class="hero-section-wrapper">
+                <div>
+                    <h3>"Hello! Leaders"</h3>
+                    <h1>THE GLOBAL <br> LEADERSHIP SUMMIT 2023</h1>
+                    <p class="announcement" id="announcement-mobile">The Global Summit brings together the community of experts, academics, activists who comprise the Graphic designs affiliate network in a different country every two years. This year, we hope to expand our invitation list including organizations and individuals</p>
+
+                    <p class="announcement" id="announcement-desktop">
+                        The Global Summit brings together the community of experts, academics, activists who comprise the Graphic designs
+                        affiliate network in a different country every two years.
+                        This year, we hope to expand our invitation list including organizations and individuals who want to work with us on shared
+                        projects that advance the cause of the graphics, free culture and open knowledge.
+                        So if you are active and engaged in the worlds of open control and knowledge-free software advocates, Wikipedia, Open Knowledge, galleries, libraries, museums, archives, governments and foundations, lawyers, and activists-we hope, you will consider joining us to build a stronger, more vibrants commons together.
+                    </p>
+
+                    <p class="contact">Please contact us per email for any further questions about Global Leadership Summit 2023</p>
+                    <a class="contact hero-email" href="mailto:chibunduonyeje1@gmail.com">chibunduonyeje1@gmail.com</a>
+                </div>
+            </div>
```

Comment:
- [ ] Also mentioned by the previous reviewer and as stated in the project requirements, kindly follow the design guideline as much as possible to make your project a close replica of the design provided. You may make sure that the background of the first section of your design has a white background. If you want that mage on the background, I can suggest you add white color on the filter

| Your Design | Template Design |
| ----------------- | --------------------- |
| ![Screenshot from 2023-01-14 22-59-56](https://user-images.githubusercontent.com/83042742/212496655-98793faf-77f1-4b33-849f-12f864f38631.png) | ![Screenshot from 2023-01-14 23-01-48](https://user-images.githubusercontent.com/83042742/212496686-0b334b66-9c2a-492b-ad0f-d89618b0e0cc.png) |